### PR TITLE
CORDOPLUG-238

### PR DESCRIPTION
**Pull request by steinwelberg**

### CODE REVIEW

**Comment by robertszuba** [comment]
Android code changes look good for me

File: src/ios/classes/OGCDVMobileAuthClient.m [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+        self.enrollCallbackId = command.callbackId;
+
+        [[ONGUserClient sharedInstance] enrollForMobileAuth:^(BOOL enrolled, NSError *_Nullable error) {
+            if (error != nil || !enrolled) {
+                [self sendErrorResultForCallbackId:self.enrollCallbackId withError:error];
+            } else {
+                [self.commandDelegate sendPluginResult:[CDVPluginResult resultWithStatus:CDVCommandStatus_OK] callbackId:self.enrollCallbackId];
+            }
+            self.enrollCallbackId = nil;
```
Comment (stnslw) [inline src/ios/classes/OGCDVMobileAuthClient.m]: It seems to me that if enroll method will be called twice, both responses will be delivered to the second callbackId stored in self. `command.callbackId` can used within the block instead, block will retain the callbackId so self does not have to.

**Comment by robertszuba** [comment]
For android automated tests are passing, so the MR can be closed as soon as iOS part is fixed

File: src/ios/classes/OGCDVMobileAuthClient.m (hunk 1 above)
Comment (steinwelberg) [inline src/ios/classes/OGCDVMobileAuthClient.m]: That is right.. I forgot to refactor that.

**Comment by steinwelberg** [comment]
For me they succeed. Please make sure that you have configured the example app with the test environment (msp-snapshot-test.onegini.io). See also the internal docs on running the tests.

**Pull request merged**